open-api/rest-catalog-open-api.yaml

```diff
@@ -960,6 +960,15 @@ paths:
           schema:
             type: string
             enum: [ all, refs ]
+        - in: query
+          name: referenced-by
```

Comment:
Can we call it something like `referenced-by-view` since it can only be referenced by a view here? I understand that the description already explicitly says that it's a view, but people can still misuse it given a generic name like `referenced by`.